docs/ML-Agents-Overview.md

```diff
@@ -350,12 +350,12 @@ training process.
   learn more about adding visual observations to an agent
   [here](Learning-Environment-Design-Agents.md#multiple-visual-observations).
 
-- **Training with Reset Parameter Sampling** - To train agents to be adapt
-  to changes in its environment (i.e., generalization), the agent should be exposed
+- **Training with Environment Parameter Randomization** - To train agents to be adapt
+  to changes in its environment (i.e., better generalize), the agent should be exposed
   to several variations of the environment. Similar to Curriculum Learning,
   where environments become more difficult as the agent learns, the toolkit provides
-  a way to randomly sample Reset Parameters of the environment during training. See
-  [Training Generalized Reinforcement Learning Agents](Training-Generalized-Reinforcement-Learning-Agents.md)
+  a way to randomly sample Environment Parameters of the environment during training. See
```

Comment:
Environment Parameters of the environment sounds weird. can "of the environment" be cut?